Source/JavaScriptCore/heap/MarkedBlock.h

```diff
@@ -425,8 +425,9 @@ class MarkedBlock {
     inline bool marksConveyLivenessDuringMarking(HeapVersion markingVersion);
     inline bool marksConveyLivenessDuringMarking(HeapVersion myMarkingVersion, HeapVersion markingVersion);
 
-    // This is only used for debugging. We should remove this once the issue is resolved (rdar://136782494)
-    NO_RETURN_DUE_TO_CRASH NEVER_INLINE void dumpInfoAndCrashForInvalidHandle(AbstractLocker&, HeapCell*);
+    // FIXME: remove these rdar://136782494) is resolved
```

Comment:
Please replace this comment with a simple `FIXME: rdar://.....` where the radar is an unresolved bug and tracks the work to clean this up.  That way, when one looks up the radar in the future, it will tell us what to clean up.

Reply:
Okay. I was just simplifying the existing comment, but I'm happy to create a new radar for this follow on cleanup.